operatorcourier/build.py

```diff
@@ -33,6 +35,10 @@ def _updateBundle(self, operatorBundle, file_name, yaml_string):
         # Determine which operator file type the yaml is
         operator_artifact = self._get_field_entry(yaml_string)
 
+        # If the file isn't one of our special types, we ignore it and return
+        if operator_artifact == "":
```

Comment:
Alternate would be to allow including all resource types that OLM supports. Please see: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/operators/catalog/operator.go#L1024

Reply:
This list is going to keep growing, I assume? Is there a reason we shouldn't just accept any yaml file?

Reply:
OLM will fail the install if it finds an unsupported resource

Reply:
It seems to me like that problem is slightly orthogonal to the issue we are trying to solve with this PR. We should address it in a followup.

Reply:
I am fine with that. Just wanted to call this out in case it was low hanging fruit.